client/js/components/map/publicdetail/controls/layerlist/DpPublicLayerListLayer.vue

```diff
@@ -100,6 +100,18 @@ export default {
     }
   },
 
+  emits: [
+    'layer:toggle',
```

Comment:
nitpick: we could consider sorting emits alphabetically

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2437/commits/8aed61aab38eb02c143f76899eef08d21555e351
Thanks!

Reply:
I think it should be like this:

```
'layer:hideOtherCategories',
'layer:showParent',
'layer:showVisibiltyGroupLayer',
'layer:toggle',
'layer:toggleLegend',
'layer:toggleOtherBaselayers',
'layer:toggleVisibiltyGroup'
'layer-opacity:change',
'layer-opacity:changed',
```

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2437/commits/15701ae32017c030618ffcbf9566159fef537f7d